apps/web/app/(auth-pages)/sign-in/page.tsx

```
@@ -97,21 +95,32 @@ export default function Login() {
 						</div>
 					)}
 
 					{authError && !isNotRegistered && (
-						<div className="text-red-600">
+						<div className="text-red-600" role="alert" aria-live="assertive">
 							There was an error during authentication. Please try again.
 						</div>
 					)}
+
+					{authSuccess && (
+						<div className="text-green-600" role="status" aria-live="polite">
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<output>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)